src/database/table.ts

```diff
@@ -30,11 +31,14 @@ export default class Table {
       `    column.table_schema = key.table_schema and`,
       `    column.table_name = key.table_name and`,
       `    column.column_name = key.column_name`,
-      `WHERE column.TABLE_SCHEMA = ? AND column.TABLE_NAME = ?`,
+      `WHERE column.TABLE_SCHEMA = ?`,
+      ...[
+        name ? `AND column.TABLE_NAME = ${name}` : ``,
```

Comment:
Please do not use the string literal `${name}` in the string and instead use the binding parameter.